spec/models/user_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../app/models/user'
+
+describe User do
+
+  let(:name) { 'Camilla' }
+  let(:username) { 'allimac' }
+  let(:email) { 'camilla@email.com' }
+  let(:password) { 'pass1234' }
+
+  it 'returns the user if the autentication is correct' do
+    user = described_class.new(name: name, username: username, email: email, password_confirmation: password)
```

Comment:
Metrics/LineLength: Line is too long. [109/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)